src/main/kotlin/no/nav/dagpenger/joark/mottak/JoarkMottak.kt

```diff
@@ -66,6 +66,7 @@ class JoarkMottak(val env: Environment, private val journalpostArkiv: Journalpos
             .mapValues(ValueMapper<GenericRecord, Behov> {
                 hentInngåendeJournalpost(it.get("journalpostId").toString())
             })
+            .selectKey { _, behov -> behov.getBehovId() }
```

Comment:
ehm, not sure about this. As we assign behov id in when fetchin and assign journal to behov we can have more than one behov id pointing to one journal post id. (otherwise we have to assure exectly once consuming). We may be better off using journal post id as partition key,.